apps/microtvm/ethosu/Makefile

```diff
@@ -19,13 +19,13 @@
 
 # Setup build environment
 BUILD_DIR := build
-STANDALONE_CRT_PATH := $(shell python3 -c "import tvm.micro; print(tvm.micro.get_standalone_crt_dir())")
 
 ARM_CPU = ARMCM55
 ETHOSU_PATH = /opt/arm/ethosu
 ETHOSU_DRIVER_PATH ?= ${ETHOSU_PATH}/core_driver
 CMSIS_PATH ?= ${ETHOSU_PATH}/cmsis
 ETHOSU_PLATFORM_PATH ?= ${ETHOSU_PATH}/core_platform
+STANDALONE_CRT_PATH := $(abspath $(BUILD_DIR))/$(shell python3 -c "import json; data=json.load(open('./build/metadata.json')); print([e for e in data['external_dependencies'] if e['short_name'] == 'tvm_standalone_crt'][0]['url'])")
```

Comment:
Do we need this to be dynamic now? I'd expect a user to just encode it as part of the demo

Reply:
I wasn't sure which way to go with this.
I've hard-coded it in the demo Makefile now, which I agree feels more natural.